src/app/public/src/modules/sidebar/sidebar-wrapper.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../styles/_variables';
+
+.stache-sidebar-wrapper {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  z-index: $stache-sidebar-z-index;
+  bottom: 0;
+  left: 0;
+  border-right: 1px solid #cdcfd2;
+  box-shadow: 0px -2px 5px 0 rgba(0, 0, 0, 0.3);
```

Comment:
The box-shadow is forcing my eyes to give the sidebar visual priority over everything else. I think we'd be okay with the thin border only.